# HG changeset patch
# User Ronan Bignaux <r.bignaux@rbi.io>
# Date 1424453230 -3600
#      Fri Feb 20 18:27:10 2015 +0100
# Node ID f40592adf09698bec0c0ba3afb9c5a2b34de9e68
# Parent  4c130dba836607004aec431040e1ee0d11eb5957
specify wxversion to avoid bug when wxpython 3 is installed

diff -r 4c130dba8366 -r f40592adf096 Beremiz.py
--- a/Beremiz.py	Mon Mar 17 16:16:06 2014 +0100
+++ b/Beremiz.py	Fri Feb 20 18:27:10 2015 +0100
@@ -25,7 +25,10 @@
 
 updateinfo_url = None
 
-import os, sys, getopt, wx
+import os, sys, getopt
+import wxversion
+wxversion.select('2.8')
+import wx
 import __builtin__
 from wx.lib.agw.advancedsplash import AdvancedSplash
 import tempfile
